src/auto/file_info.rs

```diff
@@ -233,7 +246,7 @@ impl FileInfo {
         }
     }
 
-    //pub fn set_attribute(&self, attribute: &str, type_: /*Ignored*/FileAttributeType, value_p: /*Unimplemented*/Fundamental: Pointer) {
+    //pub fn set_attribute(&self, attribute: &str, type_: FileAttributeType, value_p: /*Unimplemented*/Fundamental: Pointer) {
```

Comment:
Seems like we want some manual implementation for all this file attribute API, that safely exposes this pointer via some trait with an associated type for the different file attribute types

Reply:
For a later PR

Reply:
Just keep in mind that the API is kind of useless without

Reply:
I'll try to add it as soon as possible.

Reply:
I think the nice way to implement would be similar to the GStreamer tags support. You'd have a ZST for each attribute that implements a trait, that trait has an associated type for the attribute type and provides ways for getting from such a value to the attribute